specification/network/resource-manager/Microsoft.Network/stable/2020-05-01/networkWatcher.json

```diff
@@ -3397,10 +3397,10 @@
         },
         "context": {
           "readOnly": true,
-          "type": "array",
-          "description": "Provides additional context on the issue.",
-          "items": {
```

Comment:
This seems to be a breaking change, could you explain why not use a new API version?

Reply:
yes, @lirenhe , it's a bug fix. SDK and CLI experience is broken because of it (type is deserialized incorrectly)

Reply:
do we still need `IssueContext` ?

Reply:
no, we can't. It is used in another place. It was there before this change